Xamarin.Forms.ControlGallery.Android/Xamarin.Forms.ControlGallery.Android.csproj

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Condition="'$(EnvironmentBuildPropsImported)' != 'True'" Project="..\Environment.Build.props" />
```

Comment:
@mattleibow @Redth 

ControlGallery setup for AndroidX/Android10.0 tests look good?

Reply:
Seems to be ok.